### PR TITLE
Store logs in /var/log

### PR DESCRIPTION
**Pull request by zrhoffman**
This PR

- changes configuration to log Traffic Router to /var/log/traffic_router instead of /opt/traffic_router/var/log
- changes configuration to log Traffic Monitor to /var/log/traffic_monitor instead of /opt/traffic_monitor/var/log
- changes configuration to log Traffic Stats to /var/log/traffic_stats instead of /opt/traffic_stats/var/log

 so that the logs are on a different partition than the application itself.

<hr/>

## Which Traffic Control components are affected by this PR?

- Documentation
- Traffic Monitor
- Traffic Router
- Traffic Stats
- CDN in a Box
- Automation - Ansible roles
- unknown

## PR submission checklist
- [x] This PR has tests 
- [x] This PR has documentation 
- [x] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

**Comment by ntheanh201** [comment]
+1 this idea, it would be better for the logs agent's permissions

File: docs/source/admin/traffic_router.rst [hunk 1]
```diff
@@ -447,14 +447,14 @@ The following is an example of the request flow when a client requests the routi
 
 Troubleshooting and log files
 =============================
-Traffic Router log files can be found under :file:`/opt/traffic_router/var/log` and :file:`/opt/tomcat/logs`. Initialization and shutdown logs are in :file:`/opt/tomcat/logs/catalina{date}.out`. Application related logging is in :file:`/opt/traffic_router/var/log/traffic_router.log`, while access logs are written to :file:`/opt/traffic_router/var/log/access.log`.
+Traffic Router log files can be found under :file:`/var/log/traffic_router` and :file:`/var/log/tomcat`. Initialization and shutdown logs are in :file:`/var/log/tomcat/catalina{date}.out`. Application related logging is in :file:`/var/log/traffic_router/traffic_router.log`, while access logs are written to :file:`/var/log/traffic_routeraccess.log`.
```
Comment (ntheanh201) [inline docs/source/admin/traffic_router.rst:450]: /var/log/traffic_routeraccess.log -> /var/log/traffic_router/access.log

**Comment by jagan-parthiban** [comment]
After running Dev CIAB on the PR code couldn't find the log files either at /opt/traffic_router/var or /var/log/. Also couldn't find the log file in TM container as well. 

```
/opt/traffic_router/var # 
/opt/traffic_router/var # ls -l
total 0
drwxr-x---    4 root     root           128 Apr  4 05:19 auto-zones
/opt/traffic_router/var # cd /var/log/
/var/log # ls -l
total 0
```

**Review by jagan-parthiban — COMMENTED** [review]
After running Dev CIAB on the PR code, couldn't find the log files either at /opt/traffic_router/var or /var/log/. Also couldn't find the log file in TM container as well. 

```
/opt/traffic_router/var # 
/opt/traffic_router/var # ls -l
total 0
drwxr-x---    4 root     root           128 Apr  4 05:19 auto-zones
/opt/traffic_router/var # cd /var/log/
/var/log # ls -l
total 0
```

**Comment by zrhoffman** [comment]
> After running Dev CIAB on the PR code, couldn't find the log files either at /opt/traffic_router/var or /var/log/.

The TR logs show up for me in `/var/log/traffic_router`.

```shell
[root@trafficrouter traffic_router]# ls -l
total 76
-rw-r--r-- 1 root root     0 Apr 10 18:52 access.log
-rw-r--r-- 1 root root 73560 Apr 10 18:55 traffic_router.log
[root@trafficrouter traffic_router]# tail -f access.log
```

They also show up for `docker-compose logs -f trafficrouter`.

> Also couldn't find the log file in TM container as well.

The TM logs are printed to stdout only with no files

https://github.com/apache/trafficcontrol/blob/900cf169757b007e106ecdf112f9c520fbdd7635/infrastructure/cdn-in-a-box/traffic_monitor/traffic_monitor.cfg#L8-L13

https://github.com/apache/trafficcontrol/blob/900cf169757b007e106ecdf112f9c520fbdd7635/infrastructure/cdn-in-a-box/variables.env#L76-L81

**Review by jagan-parthiban — APPROVED** [review]

File: docs/source/admin/traffic_router.rst (hunk 1 above)
Comment (zrhoffman) [reply docs/source/admin/traffic_router.rst]: Good catch, thanks! Fixed in 72834a25bc.

**Pull request merged**